langkit/libmanage.py

```diff
@@ -97,6 +130,8 @@ def __init__(self, root_dir: Optional[str] = None) -> None:
             )
         )
 
+        self.build_mode = ""
+
```

Comment:
This is not a valid build mode, why is this needed?

Reply:
It's not, I think it's a leftover, I'll remove it.